docs/how-to-enable-new-languages.md

```diff
@@ -377,18 +377,18 @@ Then open a PR to the CDN repo to add both the YAML and Day.js files for review.
 
 ## Prep the News Repo for the New Language
 
-The [News repo](https://github.com/freeCodeCamp/news) pulls data from a Ghost instance, the files you added to the CDN, builds News, and deploys it.
+The [News repo](https://github.com/freeCodeCamp/news) pulls data from a Ghost Instance, the files you added to the CDN, builds News, and deploys it.
 
 > [!WARN]
-> Pull requests to the news repo _must_ come from the same repo. You should not work off of a fork for this step.
+> Pull requests to the News repo _must_ come from the same repo. You should not work off of a fork for this step.
```

Comment:
Welcome, @Fubge, and thank you for opening the PR.

I am not a native English speaker, so can you tell me why the other changes are typos?

I agree the "news" should be "News", but I don't see the need for the other changes.

Reply:
Hello, Sboonny, and thank you for your reply.

Both spellings can be correct, but usually when using a specific term like "Ghost interface," both words should be capitalized ("Ghost Interface"). The same applies for "Crowdin Workflows."

I definitely made a mistake with the "project" change. Not capitalizing it is the only correct way in this context. I apologize for the error and appreciate your patience.